### DISCUSSION TITLE
Use http intead of https (turn off SSL)?

### DISCUSSION THREAD

**Discussion by tsathis** [vercel/vercel#5287] [Help] [answered]
I'm very thankful for Vercel as they give a https domain. But I need to work with an API inside my app which doesn't have SSL.

i.e. My frontend is like **https**://my-app.vercel.app, but it calls an API which is like **http**://my-api.com which cause to raise a Mixed Content Error.

Is there any way to make my Vercel domain **http**?

**Comment by paulogdm** [comment, marked as answer]
There is no way to use an unsecured domain with Vercel.

**Comment by silasabbott** [comment]
For anyone else stumbling upon this question, you may be able to set [rewrites](https://vercel.com/docs/project-configuration#project-configuration/rewrites) in your `vercel.config` to access your api.

**Comment by njzydark** [comment]
You can manualy close this option:

<img width="684" alt="image" src="https://user-images.githubusercontent.com/19601720/189164620-b82c6d2d-e9ca-4712-8039-970364093490.png">

<img width="1908" alt="image" src="https://user-images.githubusercontent.com/19601720/189164409-546f82f5-d9fe-43f9-89ef-3d5cfb95ad7b.png">

**Reply by zerohk** [reply]
oh thank you wo de hxd

**Reply by omerwyo** [reply]
thank you xiongdi

**Reply by aviladiogo** [reply]
thank you

**Reply by BrunoLucass** [reply]
thks man

**Reply by ChTayyab786** [reply]
Thanks 👍

**Reply by t0ri333** [reply]
This is helpful! thanks
